scylla-cql/src/frame/value.rs

```diff
@@ -100,6 +143,21 @@ pub trait ValueList {
 
         Ok(())
     }
+
+    fn for_debug(&self) -> String {
+        let mut s = String::new();
+        self.for_debug_append(&mut s);
+        s
+    }
+
+    fn for_debug_append(&self, s: &mut String) {
+        s.push_str("<Serialized>")
+    }
+
+    fn for_debug_append_nth(&self, s: &mut String, _n: usize) -> Result<(), NotEnoughValuesError> {
+        s.push_str("<Serialized>");
+        Ok(())
+    }
```

Comment:
Similar concern here as in `Value`. Is it important to always print query arguments as a comma-separated list of string? Can we get away with just requiring `Display`?

Reply:
Besides, this is quite fragile - I think we have some public methods which take `ValueList`, call `serialized()` and then just call other methods that also take `ValueList`. If the tracing logic is situated in the latter, then we will always get "<Serialized>" in tracing and this all will be useless.

Maybe, instead, we could just print raw bytes from a serialized form? We already have `SerializedValues::iter()` that allows going over serialized values. It doesn't look as nice, but will be more consistent and less prone to such issues. Then, we wouldn't also need `Display` on `ValueList` or `Value`.

Reply:
Will the serialized form of the values be as readable and helpful in debug as the Display form is?